multinet/swagger/node_edges.yaml

```diff
@@ -0,0 +1,24 @@
+Retrieve the edges of a graph node
+---
+parameters:
+  - $ref: "#/parameters/workspace"
+  - $ref: "#/parameters/graph"
+  - $ref: "#/parameters/table"
+  - $ref: "#/parameters/node"
+  - $ref: "#/parameters/direction"
+  - $ref: "#/parameters/offset"
+  - $ref: "#/parameters/limit"
+
+responses:
+  200:
+    description: A list of edges
+    schema:
+      $ref: "#/definitions/entity_names"
```

Comment:
Can we do a richer example here?

Reply:
I think this is connected to the discussion in https://github.com/multinet-app/multinet/pull/205#discussion_r354480703.

Reply:
66e11acd1cff83b31a8fe5d1dd4315f7c525a0f7